### PR TITLE
[zh] Update storage-classes.md

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Signed-off-by: Guangwen Feng fenggw-fnst@cn.fujitsu.com

en PR #30554 
/assign @tengqm
PTAL, thanks.

### CODE REVIEW

File: content/zh/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -755,15 +755,15 @@ OpenStack 的内部驱动已经被弃用。请使用
...
 - [vCP provisioner](#vcp-provisioner): `kubernetes.io/vsphere-volume`
 
 In-tree provisioners are [deprecated](/blog/2019/12/09/kubernetes-1-17-feature-csi-migration-beta/#why-are-we-migrating-in-tree-plugins-to-csi). For more information on the CSI provisioner, see [Kubernetes vSphere CSI Driver](https://vsphere-csi-driver.sigs.k8s.io/) and [vSphereVolume CSI migration](/docs/concepts/storage/volumes/#csi-migration-5).
 -->
 vSphere 存储类有两种制备器
 
-- [CSI 制备器](#csi-provisioner): `csi.vsphere.vmware.com`
-- [vCP 制备器](#vcp-provisioner): `kubernetes.io/vsphere-volume`
+- [CSI 制备器](#vsphere-provisioner-csi): `csi.vsphere.vmware.com`
+- [vCP 制备器](#vcp-制备器): `kubernetes.io/vsphere-volume`
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-classes.md:766]: 我建议保留 vcp-provisioner 
在789行做修改： '### vCP 制备器  {#vcp-provisioner}

**Comment by fenggw-fnst** [comment]
Thanks for the review, done.

**Pull request merged**